adafruit_bluefruitspi.py

```diff
@@ -30,6 +30,7 @@
 
 import time
 import struct
+import binascii
```

Comment:
on an M0 board -- like the feather_m0_bluefruit (using the feather_m0_adalogger build), the binascii module is not compiled in so it has to be imported from the adafruit_binascii.mpy module

the simplest fails at this point on the feather_m0_bluefruit (adalogger) build.

Reply:
I modified the code to import adafruit_binascii and  the simpleteest runs OK on the feather_m0_bluefruitt with the feather)m0_adalogger build
```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:

Code done running.

Press any key to enter the REPL. Use CTRL-D to reload.

Adafruit CircuitPython 7.2.0-alpha.1-203-gbf93ddec5 on 2022-01-24; Adafruit Feather M0 Adalogger with samd21g18
>>> import bluefruitspi_simpletest
Initializing the Bluefruit LE SPI Friend module
BLESPIFRIEND
nRF51822 QFACA10
78585487A8785672
0.8.1
0.8.1
Apr 10 2019
S110 8.0.0, 0.2

Waiting for a connection to Bluefruit LE Connect ...
...................................................
 *Connected!*
```

Perhaps a try/except can be used to select the proper import